app/javascript/packs/articleModerationTools.js

```diff
@@ -12,20 +13,16 @@ const initializeModerationsTools = async () => {
   const { initializeFlagUserModal } = await import('./flagUserModal');
 
   // article show page
-  if (
-    user?.trusted &&
-    user?.id !== articleAuthorId &&
-    !top.document.location.pathname.endsWith('/mod')
-  ) {
-    initializeActionsPanel(user, path);
-    initializeFlagUserModal(articleAuthorId);
-    // dev.to/mod
-  } else if (user?.trusted && top.document.location.pathname.endsWith('/mod')) {
-    initializeActionsPanel(user, path);
-    initializeFlagUserModal(articleAuthorId);
+  if (user?.trusted) {
+    if (user?.id !== articleAuthorId && !isModerationPage()) {
+      initializeActionsPanel(user, path);
```

Comment:
I'm probably missing something, but why is the contents of the `if` statement and `else if` statement the same? 

```
initializeActionsPanel(user, path);
initializeFlagUserModal(articleAuthorId);
```

Can we not modify this into one statement?

Reply:
@Ridhwana, it previously had the `user?.trusted` as part of the condition for the `if` and the `if/else`. Now, we only enter the inner `if/else` if it's a trusted user.